public/javascripts/EditCartItems.js

```diff
@@ -0,0 +1,290 @@
+/**
+ * Created by johnschroeder on 6/23/15.
+ */
+
+//TODO: make sure the database hit works once stored procedures stop timing out.
+//TODO: comment!
```

Comment:
Can we add a usage here saying what args this page expects like we do with routes on the backend?